src/javascript/app/pages/form/form-component.jsx

```diff
@@ -191,15 +191,15 @@ export const FormComponent = () => {
                                             options={start_dates.options}
                                             value={findTextByValue(start_dates.options, date_start)}
                                             onUpdate={updateFormField}
-                                            elementId='date_start'
+                                            element_id='date_start'
```

Comment:
please help to update the `elementId ` prop to `element_id` in DropdownComponent as well. This can break the functionality.

Reply:
For `DropdownComponent` and `NumbersDropdown` I kept it `elementId` because it's a prop.

Reply:
@prince-deriv I think you missed to revert the change here.

Reply:
@Nuzhy-Deriv  please check one more time